app/assets/v2/js/grants/detail.js

```diff
@@ -86,24 +91,35 @@ $(document).ready(function() {
 
     let edit_title = $('#form--input__title').val();
     let edit_reference_url = $('#form--input__reference-url').val();
-    let edit_admin_profile = $('#grant-admin option').last().text();
-    let edit_description = grant_description.getText();
-    let edit_description_rich = JSON.stringify(grant_description.getContents());
+    let edit_admin_profile = $('#grant-admin option')
+      .last()
+      .text();
```

Comment:
^ could we throw those two into L94 as opposed to breaking it ?

Reply:
Done: https://github.com/gitcoinco/web/pull/4793/commits/a5a190018ba89f013f0d16880820a89a85b706eb
Must have been formatted by my Prettier plugin